tests/test_handler_mlflow.py

```diff
@@ -61,6 +86,18 @@ def _update_metric(engine):
             # check logging output
             self.assertTrue(len(glob.glob(test_path)) > 0)
 
+    def test_multi_thread(self):
+        test_uri_list = ["monai_mlflow_test1", "monai_mlflow_test2"]
+        with ThreadPoolExecutor(2, "Training") as executor:
+            futures = {}
+            for t in test_uri_list:
+                futures[t] = executor.submit(dummy_train, t)
+
+            for _, future in futures.items():
+                res = future.result()
+                self.assertTrue(len(glob.glob(res)) > 0)
+                shutil.rmtree(res)
```

Comment:
will this leave unclosed temporary files/folders if the test fails?

Reply:
Hi @binliunls ,

Please refer to other test program to set the `tempdir`, for example:
https://github.com/Project-MONAI/MONAI/blob/dev/tests/test_lmdbdataset_dist.py#L34-L38

Thanks.

Reply:
Hi @Nic-Ma ,
      I have added a try finally logic. Is it ok for this issue.

Thanks,
Bin